src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -84,39 +84,6 @@ def get_registry_by_name(cli_ctx, registry_name, resource_group_name=None):
     return client.get(resource_group_name, registry_name), resource_group_name
 
 
-def arm_deploy_template_managed_storage(cli_ctx,
```

Comment:
Why is this being removed?  Is this a breaking change?

Reply:
We wanted to switch to use an SDK PUT request to create the resource rather than using an ARM template deployment. Not a breaking change.